cdap-ui/templates/apps/predefined/AmazonSQSToHBase.json

```diff
@@ -28,6 +28,7 @@
     "transforms": [],
     "instances": 1
   },
-  "description": "Realtime updates from Amazon Simple Queue Service to HBase."
+  "description": "Real-time updates from Amazon Simple Queue Service into an HBase table",
```

Comment:
We only use "Realtime" when it is the actual name (such as an artifact or plugin name, or portion). Otherwise, we follow the rules as described by @jwang47 .